### DISCUSSION TITLE
How to insert metatags to my site/book (like in metathis)?

### DISCUSSION THREAD

**Discussion by jtrecenti** [quarto-dev/quarto-cli#3148] [Q&A] [answered]
Hi! I'm writing a book using Quarto, and so far, it has been a great experience 🎉 Thank you for all the effort.

https://livro.abj.org.br/

Now I need to improve the metadata of the site. Looking at the documentation, I found this: https://quarto.org/docs/reference/formats/html.html#metadata

I also looked at the YAML file from R4DS: https://github.com/hadley/r4ds/blob/main/_quarto.yml

However, I didn't find any easy way to insert other metatags, such as images and other stuff. With RMarkdown, I used to accomplish that using [`{metathis}` package](https://pkg.garrickadenbuie.com/metathis/).

 Any tips? Thanks!

PS: I usually test my metatags with https://metatags.io/

**Comment by dragonstyle** [comment, marked as answer]
You can also provide metadata for Twitter / Opengraph (more here: https://quarto.org/docs/websites/website-tools.html#social-metadata) via YAML or could use `include-in-header` to directly place meta tags in the head of the document (https://quarto.org/docs/output-formats/html-basics.html#includes).

HTH!
